core/src/main/kotlin/org/evomaster/core/output/service/TestCaseWriter.kt

```diff
@@ -88,14 +93,66 @@ abstract class TestCaseWriter {
         return lines
     }
 
+    protected fun handleExternalServiceActions(
+        lines: Lines,
+        actions: List<HttpExternalServiceAction>
+    ) {
+        // TODO: Handle same request pattern exists multiple time.
+        //  Currently we don't handle the same request pattern exists multiple
+        //  time. For that, need to create WireMock scenario to have multiple
+        //  responses for the same request pattern.
+        actions
+            .filter { action -> action.active }
+            .forEachIndexed { index, action ->
+                val response = action.response as HttpWsResponseParam
+                val name = getWireMockVariableName(action)
+
+                // Default behaviour of WireMock has been removed, since found no purpose
+                // in case if there is a failure regarding no routes found in WireMock
+                // consider adding that later
+                lines.add("assertNotNull(${name})")
+                lines.add("${name}.stubFor(")
+                lines.indented {
+                    lines.add(
+                        "${
+                            action.request.method.lowercase()
+                        }(urlEqualTo(\"${action.request.url}\"))"
+                    )
+                    // adding priority from the index of the respective action
+                    // TODO: when handling multiple calls need to fix this
+                    lines.add(".atPriority(${index + 1})")
+                    lines.add(".willReturn(")
+                    lines.indented {
+                        lines.add("aResponse()")
+                        lines.indented {
+                            lines.add(".withStatus(${response.status.getValueAsRawString()})")
+                            // TODO: Response is empty for now
+                            lines.add(".withBody(\"${response.response.getValueAsRawString()}\")")
+                        }
+                        lines.add(")")
+                    }
+                }
+                lines.add(")")
+                if (format.isJava()) {
```

Comment:
why the check on `format.isJava()`?

Reply:
From the previous implementation. Forget to remove it.